bot.go

```diff
@@ -79,3 +81,6 @@ type Bot interface {
 	// action queue and it will be processed asynchronously.
 	InvokeAction(id string, user User, action UserAction)
 }
+
+// Sendable is the interface that must implements structs that are being sent by a Bot
+type Sendable fmt.Stringer
```

Comment:
why would something Sendable be a stringer? They have nothing to do with a string. Also, everything with a `String` method will be sendable, which is the opposite of what we want.

we want something like

```go
type Sendable interface {
	isSendable()
}

func (Foo) isSendable() {}

func (Bar) isSendable() {}
```